components/application/application.js

```diff
@@ -250,6 +251,19 @@ async function adminPatchRejectApplication(req,res,next){
     }
 }
 
+async function getPaymentSize(req, res, next) {
+    // This method calculates the payment sizes based on the requested loan amount from 2 -> 12 and returns values
+    const { requestedLoanAmount } = req.query;

```

Comment:
This could be a param so that if the request is made without it, it isn't recognized